### ISSUE TITLE
Relicense to GPLv3+

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/ncdns-nsis#84]
I would like to relicense this repo from MIT to GPLv3+.  If there are any objections, please post in this issue within 1 month.

**Comment by JeremyRand** [comment]
CC: @domob1812 @hlandau

**Comment by JeremyRand** [comment]
ncdns itself is already GPLv3+.  AFAIK the viral nature of the GPL cannot traverse from an NSIS script to the files installed by that NSIS script, since this does not constitute binary linking.  So non-GPLv3+ software that we might want to install, e.g. ncp11, is not impacted.

**Comment by hlandau** [comment]
I don't think you need my permission for this since this is a move from a less restrictive to a more restrictive license, but you have my permission anyway. "GPLv3 or later". This goes for ncdns also.

**Comment by domob1812** [comment]
No objections from my side
